server/server.js

```diff
@@ -0,0 +1,49 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const path = require('path');
+const router = require('./routes/router.js');
+
+const app = express();
+const port = 3050;
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(express.json());
+
+app.use(express.static(path.join(__dirname, '..', 'public')));
+
+app.get('/', (_, response) => {
+  response.send(express.static(__dirname, '..', 'public', 'index.html'));
+});
+
+// Finds a /########## string, where # => any digit
+app.get(/(?:\/\d{10})(?!.)/, (request, response) => {
+  router.getSong(request.url.match(/(?:\d{10})/), (song) => {
+    console.log('The song requested was id: \n', song[0]);
+    response.send(song);
```

Comment:
Be sure to add in some error handling, making use of response.status to send a relevant HTTP status code

Reply:
I have opted to leave this as-is for now. The way it is currently built it isn't really possible for this function interaction to create an error that comes from the server or client. Rather, the only error possible here is that the path is not properly reached (error handling performed by express) or the database request comes back faulty (handled by the database).

As far as the status codes,  I definitely could find something more relevant versus relying on the inherent 200 status, but at this time it does not appear to be essential.